terminus/generators/monolane_generator.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import print_function
+
+from collections import OrderedDict
+from math import atan2
+from math import degrees
+import yaml
+
+from .city_visitor import CityVisitor
+from models.intersection_node import IntersectionNode
+
+
+class MonolaneGenerator(CityVisitor):
```

Comment:
It would be great to have a couple of simple tests to understand what we expect this class to generate. You can use the same samples in `RNDFGeneratorTest` if you want (now that I'm thinking, it would be great to use the same examples, we can later factor them out and use them as the standard tests for all the generators).